rpi/controller.py

```diff
@@ -200,41 +186,43 @@ def doorcheck_by_rules(self, string):
 # It runs all the time during normal operation
 def run():
     b = Board()
-
-    log("started.")
+    l = Logger(5)#put a number from 0-5 here, if the logs are getting too large, try lowering this
```

Comment:
I'd like to move away from this old-style naming convention.
This was useful when program size was at a premium, but it's been a good long while since that was a concern.